components/fxa-client/src/lib.rs

```diff
@@ -239,6 +239,8 @@ impl FirefoxAccount {
     }
 }
 
+#[derive(Debug, Serialize)]
+#[serde(tag = "event_type", content = "data")]
```

Comment:
Do you want to `rename_all = "camelCase"`, too, or does that only affect the variant names? (So we'll have `{ event_type: "IncomingDeviceCommand", data: { command_type: "TabReceived", data: { sender: ... } } }` instead of `incoming_device_command` and `tab_received`)?

Reply:
That's a great idea! I'll add that :)